lib/rmg/CRmgTemplate.h

```diff
@@ -41,17 +41,24 @@ namespace EWaterContent
 	};
 }
 
-namespace EMonsterStrength
+namespace EZoneMonsterStrength // used as int in monster generation procedure
```

Comment:
can we have only one enum?

Reply:
I'll change RANDOM = -1 back to RANDOM = -2 in EGlobalMonsterStrength, update the branch with recent changes from develop and see if that solves the problem. And if not, I'll investigate further.
As for one enum, the thing is that EZoneMonsterStrength::RANDOM wouldn't make sense while EGlobalMonsterStrength::RANDOM does make sense and the upcoming EZoneMonsterStrength::NONE will make sense (Hota rmg has that option) while EGlobalMonsterStrength::NONE wouldn't make sense.
And it would be nice if adding parsing of "monsters": "none" to ZoneOptions::serializeJson in lib/rmg/CRmgTemplate.cpp didn't make the code more complicated.

Reply:
But would be possible to have single enum with both values?
RANDOM, NONE, WEAK, NORMAL, STRONG

Than change formula and that's it?

Reply:
I've changed my mind. I do want both zoneMonsterStrength "random" (randomly chosen for each monster) and perhaps even globalMonsterStrength "none" (though not sure if anyone would be using that), so I'll be going back to 1 enum and implementing both none and random functionality for zones and maybe also the none global functionality. Though not sure about the last one.
I'm also wondering if global monster strength: "random" should mean: "pick one strength at random at map generation start" or "pick one at random every time you make a monster on a map". I mean, it used to work like the first one, but not sure if anyone was using it and the second one is an interesting option I think. Hmm.. would be too chaotic with the none options, so I think the "random" options shouldn't pick the "none" options.

Reply:
To pick random strength every time you put a monster, this would make random maps even more, luck based. Do not think such an option is needed at the moment. (other opinions?). Random at map start looks ok to me.